metaci/release/templates/cohort/detail.html

```diff
@@ -0,0 +1,108 @@
+{% extends 'layout_full.html' %}
+
+<!-- TODO: link back to /cohorts -->
```

Comment:
I haven't forgot about this.

Reply:
Or 💭 ... maybe the story for adding navigation to the list view could include making a "Releases" tab on MetaCI's header that only people with the view cohorts perm could see.